File: Dante1/Module1.07/Zadanie7.09/main.c
#include <stdio.h>

float poly(float);

int main(void)
{
    float number;
    printf("Enter float: ");
    int check = scanf("%f",&number);
    if(check != 1)
    {
        printf("Incorrect input\n");
        return 1;
    }
    printf("Result is: %.2f",poly(number));
    return 0;
}

float poly(float num)
{
    return 5 * num * num + 12.55 * num + 0.75;
}

File: Dante1/Module1.07/Zadanie7.09/main.rs
fn poly(number: f32) -> f32 {
    5.0 * number * number + 12.55 * number + number + 0.75
}

fn main() {
    let mut input: String = String::new();
    println!("Enter: ");

    std::io::stdin().read_line(&mut input).expect("err");
    let number: f32 = input.trim().parse().expect("err");

    println!("Result is: {:.2}", poly(number));
}
